### DISCUSSION TITLE
LLama3.1 and Open WebUI on MacBook Pro [Help]

### DISCUSSION THREAD

**Discussion by JoeZhou59** [open-webui/open-webui#5203] [Q&A]
Hi everyone,
I am currently a college student at US majoring in stats. Previously, I saw a post showing how to download llama3.1 7b at Ollama and set on Mac Terminal, together with Open WebUI. I run Ollama and downloaded Docker and then runt the code under "Installing Open WebUI with Bundled Ollama Support - For CPU Only". However, as I open the link on docker 3000:8000, it says there is no model found. How can I solve it? Thanks!
<img width="892" alt="Screenshot 2024-09-05 at 11 54 07 PM" src="https://github.com/user-attachments/assets/3abbddd0-7897-42cd-8989-5dab0d4028cd">

**Comment by CSAnetGmbH** [comment]
Hi, open WebUI doesn´t work with apple. I have tried to get to work, but when Ollama find the Model the next Problem ist, when you chat with the llm it does not response you will see only Lines.

**Reply by JoeZhou59** [reply]
Thanks buddy. It turns out WebUI allows users to download models at there. I tried it several times and eventually worked! Tried Qwen2 successfully as well.
